### DISCUSSION TITLE
Unable to merge pull request

### DISCUSSION THREAD

**Discussion by candidexmedia** [community/community#31262] [Pull Requests]
Hello!  

I'm trying to [merge a pull request](https://github.com/havecookie/haveacookie/pull/6), but the merge pull request button is missing.

Instead, I've been getting this loading icon for the past 20 mins:

![image](https://user-images.githubusercontent.com/95003834/187289973-73eb4d3a-f530-44aa-954e-bf2fc28aa309.png)

I've tried Firefox, Chrome, etc. but nothing seems to help

**Comment by queenofcorgis** [comment]
Hi there! 

Are you still experiencing this? And is it just in that repository?

**Reply by candidexmedia** [reply]
It was only this repo, and I was eventually able to merge it, but it took a few hours....

**Comment by Kirtishukla2004** [comment]
hey ,
go on the link below to find the solution 
https://devopspilot.com/content/git/tutorials/11-how-to-fix-merge-conflict-in-pull-request.html#:~:text=Click%20on%20Merge%20pull%20request%20%E2%80%93%3E%20Confirm%20merge,with%20merge%20conflict%20and%20Click%20on%20Resolve%20conflicts
**hope it will help **

**Reply by candidexmedia** [reply]
Thank you for sharing!

**Reply by Kirtishukla2004** [reply]
Glad I could help! Marking the answer would be appreciated. 😸
